features/step_definitions/civil_journey_steps.rb

```diff
@@ -942,12 +948,23 @@
 end
 
 Then("the answer for {string} should be {string}") do |field_name, answer|
+  # TODO: Deprecate or refactor this so it actually works as expected
```

Comment:
nit, TODO: handle, ticket, remove?

Reply:
Ticket is raised, I'll add a refernce to the comment :+1: